script/storage.md

```diff
@@ -1,22 +1,66 @@
 ## Storage
 
-There is a persistent storage object that you must use in case you want to keep some data from the user or other things.
-This data will be loaded again when the user loads a saved game.
+The storage is an object in which we can put everything inside we want to save when a player saves the game.
+
+We have to create a property where the player can store his data in it. So in `storage.js` we start with...
 
 ```javascript
 var storage = {
+  player: { // add property
+    name: "" // add placeholder
+  }
+}
 
+// Syntax
+var storage = { // object
```

Comment:
Maybe we should indent all comments to be aligned so it's more readable. However it seems GitHub makes some weird stuff about spaces and indenting them correctly is kind of weird so we would need a way to correct that.

```javascript
var storage = {           // object
    property#1: {         // property of object
    placeholder#1: 0,     // integer of property
    placeholder#2: ""     // string of property
  }
 }
```
Also, I understand the reasoning behind the numbers (i.e. the #1) but could it result confusing for people if they actually tried to name their properties using the `#`? It is not a valid character in a JSON key (unless it's a string) so if they try it like that, it will bring up an error.